### PR TITLE
Henry

### PR DESCRIPTION
**Pull request by aesuan**

### CODE REVIEW

File: src/components/ScribbleBoard.jsx [hunk 1]
```diff
@@ -35,15 +44,21 @@ function ScribbleBoard({ socket, currentDrawer }) {
...
+      let color = isDraw ? 0 : 255;
+
+      //draw line
+      p5.stroke(color);
+      p5.strokeWeight(4);
+      p5.line(p5.mouseX, p5.mouseY, p5.pmouseX, p5.pmouseY);
+
       // Send the mouse coordinates
-      sendmouse(p5.mouseX, p5.mouseY);
+      sendmouse(p5.mouseX, p5.mouseY, p5.pmouseX, p5.pmouseY, isDraw);
```
Comment (aesuan) [inline src/components/ScribbleBoard.jsx:55]: cool! line coordinates! good job

File: src/components/ScribbleBoard.jsx [hunk 2]
```diff
@@ -2,29 +2,38 @@ import React from "react";
...
 
-function ScribbleBoard({ socket, currentDrawer }) {
+function ScribbleBoard({ socket, currentDrawer, isDraw }) {
   let x = 50;
   const y = 50;
 
   const setup = (p5, canvasParentRef) => {
     p5.createCanvas(500, 500).parent(canvasParentRef);
-    p5.background(0);
+    p5.background(255); //white
```
Comment (aesuan) [inline src/components/ScribbleBoard.jsx:11]: is so you can erase using white?

File: src/components/ScribbleBoard.jsx [hunk 3]
```diff
@@ -2,29 +2,38 @@ import React from "react";
...
-      p5.fill(0, 0, 255);
-      p5.noStroke();
-      p5.ellipse(data.x, data.y, 20, 20);
+      let color = data.isDraw ? 0 : 255;
+      p5.stroke(color);
+      p5.strokeWeight(4);
+      p5.line(data.x, data.y, data.px, data.py);
     });
+
+    socket.on(SocketEvents.CLEAR_DRAW, function() {
```
Comment (aesuan) [inline src/components/ScribbleBoard.jsx:21]: looks good!

**Review by aesuan — COMMENTED** [review]
Please remove tracking from node modules! transpiled files in public should also be gitignored in the future.

**Review by karebae — APPROVED** [review]

**Review by karebae — APPROVED** [review]
looks good

**Pull request merged**